openedx/features/course_experience/static/course_experience/js/CourseOutline.js

```diff
@@ -0,0 +1,37 @@
+/* globals Logger */
```

Comment:
If I'm understanding it correctly, this is the equivalent file to the old factory, but written in a more modern way. Is the intention that this class represents a single view, or is it the code to set up the page? I'd prefer it be the former, and that the constructor take the element that this view should attach to. That would then allow this fragment to be rendered multiple times per page (probably not important for the course outline, but it would be a good practice to support this in general).